### ISSUE TITLE
[Bug] <can not export a pdf resume ,a about:blank window pops up every time I click the issue is not working for me>

### ISSUE THREAD

**Issue by Rickustc** [AmruthPillai/Reactive-Resume#1979]
### Is there an existing issue for this?

- [X] Yes, I have searched the existing issues and none of them match my problem.

### Product Variant

Cloud (https://rxresu.me)

### Current Behavior

can not export a pdf resume ,a about:blank window pops up every time I click the issue is not working for me

### What template are you using?

None

**Comment by Rickustc** [comment]
win 11 + firefox /edge /chrome has been tried

**Comment by Normanras** [comment]
Just confirmed that this is happening for me as well, but I'm running a self-hosted version without a FQDN. Not sure if that matters (didn't see anything in docs), but just sharing for viz.

**Comment by nate235** [comment]
Yep, same issue here on self-hosted. Is there a way around this? Of course this happens when I need to send someone a resume lmao

**Comment by vietanhdev** [comment]
Same issue here.

**Comment by 00TaciTa00** [comment]
Does your resume have non-English or non-numeric characters in the title? In my case, the same thing happened when I titled it in Korean. I changed it to English and it worked.
- Translated by translator.

**Comment by YosypC** [comment]
After having decided to manually rebuild it again, character by character, I attempted to import the JSON from the broken file. I can say not only the JSON being valid, but it successfully imported it and make a working PDF.

So, the workaround is exporting a JSON (which you should always have), and importing it again.

**Comment by Normanras** [comment]
@YosypC I tried this thinking maybe it would crazily work... sadly it didn't. Are you (and same question to you, @00TaciTa00 !) on the cloud or self-hosted version? I don't _think_ it should matter, but just want to check.

**Comment by YosypC** [comment]
@Normanras web

**Comment by vietanhdev** [comment]
@YosypC @Normanras
I resolved my issue by fixing the network connection from inside Docker.
- `docker exec` into chromium container.
- Check if you can `wget` your web.
- Check and fix the firewall / Docker config if you cannot `wget`.

**Comment by Normanras** [comment]
@vietanhdev What do you mean by 'wget your web'? I can see that the chromium container is on the same network with the app/s3/db containers. Obviously, no port associated with that container as the compose file doesn't indicate this either.

**Comment by vietanhdev** [comment]
In my deployment, I separated them so they are not on the same machine.

**Comment by JackTyson** [comment]
I was able to fix this issue by doing the following:

1. Set the network_mode option to "host" for all containers.
2. Change the default port for the RxResume app to 3001 (3000 is used by chrome).
3. Modify the addresses in the docker compose file to reference localhost (or 127.0.0.1) as the address, instead of the container hostname. For example, change the lines "DATABASE_URL: postgresql://postgres:postgres@postgres:5432/postgres" to "DATABASE_URL: postgresql://postgres:postgres@127.0.0.1:5432/postgres" and "CHROME_URL: ws://chrome:3000" to "CHROME_URL: ws://127.0.0.1:3000".

I have not had an issue with it since changing the configuration.

**Comment by 00TaciTa00** [comment]
@Normanras 
A little late, web.

**Comment by amruthpillai** [comment]
Closing this issue as I'm doing some clean up to move to a newer version of the project. I apologize in advance for the inconvenience if you find the need to reopen this issue.

Please read #2499 for more information about the migration.
